src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import './Calculator.css';
+import MyCalculator from './components/Calculator';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <MyCalculator />
   </React.StrictMode>,
```

Comment:
- [ ] You don't need to remove the `App` component even though you have only a single child component Calculator. For the sake of `React/JS best practice`, you should always use the App component as a wrapper. Kindly bring back `App.js` and render the `MyCalculator` component inside it. :pray:
- [ ] Also make sure that the `App.js` component should be class-based as per the project's requirements :+1: